test/simple.test.ts

```diff
@@ -67,14 +69,21 @@ integrationTest("finish signal is handled", async () => {
 });
 
 integrationTest("mapping an extensionless CommonJS file", () => {
-  expect(runAppmapNode("./extensionless").status).toBe(0);
+  const args = ["./extensionless"];
+  if (process.platform == "win32") args.unshift("node");
+  expect(runAppmapNode(...args).status).toBe(0);
   expect(readAppmap()).toMatchSnapshot();
 });
 
 integrationTest("running a script after changing the current directory", () => {
   // Need to make sure the appmap "root" stays the same after
   // appmap-node is run, even if the current directory changes.
-  expect(runAppmapNode("bash", "-c", "cd subproject; node index.js").status).toBe(0);
+  const args =
+    process.platform == "win32"
+      ? ["cd subproject & node ../index.js"]
```

Comment:
I'm not convinced if this does what we need it do do. Especially given that `..`. (Perhaps instead of the symlink there should be a simple script in `subproject`, named differently from `index` to avoid confusion.)